staging/src/k8s.io/cluster-bootstrap/OWNERS

```diff
@@ -0,0 +1,19 @@
+approvers:
```

Comment:
these are a lot of owners. Who is the real owner of this? Would expect more like 3 people.

Reply:
sig-cluster-lifecycle

Reply:
removed all others, replaced with sig-cluster-lifecycle-leads